lib/twilio.rb

```diff
@@ -0,0 +1,21 @@
+require 'rubygems'
+require_relative 'customer'
+
+class Twilio
+  def timer
+    date = Time.now + (3600)
+    delivery_time = date.to_s.split(' ')[1]
+  end
+end
+
+def send_text(name, phone)
+  account_sid = 'AC2f4d6a3fb2fcdd3e058b6930b7f47697'
+  auth_token = '18a596c4b02371cb565004ab54694bcf'
+  @client = Twilio::REST::Client.new account_sid, auth_token
+
+  message = @client.account.messages.create(
+  :body => "#{name}Your food is on its way! It will be there by #{Twilio.timer}" ,
+  :to => phone,
+  :from => '+441767512044')
+  puts "#{name}, we just sent you a text! Food will be there by #{Twilio.timer}!"
```

Comment:
Line is too long. [81/80]<br>Do not write to stdout. Use Rails' logger if you want to log.